commit c05f00666c730de95a2e18f75326d3e9ee128bdd
Author: Michal Prívozník <mprivozn@redhat.com>
Date:   Thu Jan 28 14:08:24 2021 +0100

    conf: Drop empty virDomainNetDefPostParse()
    
    The previous commit rendered this function empty and needless.
    Remove it.
    
    Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
    Reviewed-by: Ján Tomko <jtomko@redhat.com>

Index: libvirt-7.0.0/src/conf/domain_conf.c
===================================================================
--- libvirt-7.0.0.orig/src/conf/domain_conf.c
+++ libvirt-7.0.0/src/conf/domain_conf.c
@@ -5274,13 +5274,6 @@ virDomainControllerDefPostParse(virDomai
 }
 
 
-static int
-virDomainNetDefPostParse(virDomainNetDefPtr net G_GNUC_UNUSED)
-{
-    return 0;
-}
-
-
 static void
 virDomainVsockDefPostParse(virDomainVsockDefPtr vsock)
 {
@@ -5329,10 +5322,6 @@ virDomainDeviceDefPostParseCommon(virDom
         ret = virDomainControllerDefPostParse(dev->data.controller);
         break;
 
-    case VIR_DOMAIN_DEVICE_NET:
-        ret = virDomainNetDefPostParse(dev->data.net);
-        break;
-
     case VIR_DOMAIN_DEVICE_VSOCK:
         virDomainVsockDefPostParse(dev->data.vsock);
         ret = 0;
@@ -5340,6 +5329,7 @@ virDomainDeviceDefPostParseCommon(virDom
 
     case VIR_DOMAIN_DEVICE_LEASE:
     case VIR_DOMAIN_DEVICE_FS:
+    case VIR_DOMAIN_DEVICE_NET:
     case VIR_DOMAIN_DEVICE_INPUT:
     case VIR_DOMAIN_DEVICE_SOUND:
     case VIR_DOMAIN_DEVICE_WATCHDOG:
